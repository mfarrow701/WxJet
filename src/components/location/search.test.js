import React from 'react';
import {shallow} from 'enzyme';
import Search from './search';

describe('Search component', () => {
    let component;

    const renderWrapper = () => {
        component = shallow(<Search/>);
    };

    beforeEach(() => {
        renderWrapper();
    });

    it('should render the search component after load', () => {
        expect(component).toMatchSnapshot();
    });
});
