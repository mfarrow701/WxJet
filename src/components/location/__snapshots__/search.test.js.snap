// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component should render the search component after load 1`] = `undefined`;
